src/genie/libs/parser/nxos/tests/ShowHsrpEventHistoryDebugs/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,144 @@
+expected_output = {
+    "debug_logs": {
+        "1": {
```

Comment:
Anywhere there will only ever be an integer, it should be defined as an integer in the schema and then inserted into the dictionary as an integer. Same for `pid` for example.